src/main/kotlin/app/batch/JobCompletionNotificationListener.kt

```diff
@@ -100,6 +101,13 @@ class JobCompletionNotificationListener(
         }
     }
 
+
+    private fun sendRisJsons(completionStatus: ExportCompletionStatus) {
```

Comment:
Wasn't this method name temporary until we could think of a better one-  sendPdmCommonModelMessage or something like that?